db/seeds.rb

```diff
@@ -191,11 +716,16 @@
       .find_or_create_by!(name: "")
       .update!(legacy: false, application_form_enabled: true)
   else
-    regions.each do |name|
-      country
-        .regions
-        .find_or_create_by!(name:)
-        .update!(legacy: false, application_form_enabled: true)
+    regions.each do |region|
+      next if country.regions.where(name: region[:name]).any?
+
+      country.regions.create!(
```

Comment:
```suggestion
      country.regions.create!(region)
```

Looks like we don't need to pass the individual hash values here.

Reply:
That is a very good point... Fixed in [a2903ed](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/367/commits/a2903ede7ca086e13cf16ca965478e4b8d07f200)